tools/clean_webconsole.py

```diff
@@ -23,7 +23,7 @@ def nice(x):
 
 
 repdata = (
-  Path(path_from_root('system/include/GL/gl.h')).read_text().splitline(keepends=True) +
+  Path(path_from_root('system/include/GL/gl.h')).read_text().splitlines(keepends=True) +
```

Comment:
This is an example of real typo (most likely in dead code) found by the type checker.